src/evict/evict_page.c

```diff
@@ -663,34 +663,62 @@ __evict_review(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t evict_flags, bool
     /* Make sure that both conditions above are not true at the same time. */
     WT_ASSERT(session, !use_snapshot_for_app_thread || !is_eviction_thread);
 
-    if (!conn->txn_global.checkpoint_running && !WT_IS_HS(btree->dhandle) &&
-      (use_snapshot_for_app_thread || is_eviction_thread)) {
+    /* History store data are always evictable. */
```

Comment:
Grammar: "History store data is always evictable."

Reply:
I thought the word data is plural. Found an interesting article discuss this https://ludwig.guru/blog/data-is-vs-data-are-an-intriguing-dilemma-with-a-surprising-twist/?gclid=CjwKCAjw_o-HBhAsEiwANqYhp_o85TtjLxJIIq11Ux4M1cvCw7lrhc7_rwGYn95QM-Zpy3N6swgkLRoCojEQAvD_BwE

Reply:
Huh fair enough, at first glance it does seem wrong to me so I guess the "data is" that i've come to expect is a modernization? I think we should go with the more modern form of "is" still but you've proven that what you wrote is correct anyway so your choice.

Reply:
OK, I'll go for the popular one.